android/build.gradle

```diff
@@ -51,16 +55,23 @@ android {
             }
         }
     }
+
+    // TODO: Enable when issues with AGP 7.1+ are resolved (https://github.com/GodotVR/godot_openxr/issues/187).
+//    publishing {
+//        singleVariant("fullRelease") {
+//            withSourcesJar()
+//            withJavadocJar()
+//        }
+
+    //singleVariant("stubRelease") {
+//            withSourcesJar()
+//            withJavadocJar()
+//        }
+//    }
 }
 
 dependencies {
-    if (rootProject.findProject(":godot:lib") != null) {
-        // Used for development and debugging
-        compileOnly(project(":godot:lib"))
-    } else {
-        compileOnly(fileTree(dir: "libs", include: ["godot-lib*.aar"]))
-    }
-
+    implementation "io.github.m4gr3d:godot:${versions.godotVersion}"
```

Comment:
Should this me m4gr3d? not GodotVR once merged?

Reply:
This is the dependency for the Godot Android library. For now until https://github.com/godotengine/godot-proposals/issues/4230 is completed, I'm using my own sonatype account hence why it's `io.github.m4gr3d`.

Once https://github.com/godotengine/godot-proposals/issues/4230 is complete, it will be switched to something like `org.godotengine:godot:<version>`.

Reply:
fyi, I've added https://github.com/GodotVR/godot_openxr/issues/191 to track the remaining steps.